tests/components/utils-test.js

```diff
@@ -527,47 +527,164 @@ describe('Utils', function() {
     expect(isIE10).toBeFalsy();
   });
 
-  it('tests getLanguageToUse', function() {
-    var skinConfig = {
+  describe('tests getDefaultLanguage', function() {
+    it('should return an empty string if param "localization" is wrong type', function(){
+      const emptyToParam = Utils.getDefaultLanguage();
+      expect(emptyToParam).toBe('');
+      const nullToParam = Utils.getDefaultLanguage(null);
+      expect(nullToParam).toBe('');
+      const arrayToParam = Utils.getDefaultLanguage([1, 2, 3]);
+      expect(arrayToParam).toBe('');
+    });
+
+    it('should return an empty string if param "localization" does not have field "defaultLanguage',
+      function() {
+        const defaultLanguage = Utils.getDefaultLanguage({'test': 1});
+        expect(defaultLanguage).toBe('');
+      }
+    );
+
+    it('should return "defaultLanguage" from "localization" object', function(){
+      const defaultLanguage = Utils.getDefaultLanguage({'defaultLanguage': 'en'});
+      expect(defaultLanguage).toBe('en');
+    });
+  });
+
+  describe('tests getLanguageToUse', function() {
+    let skinConfig = {
```

Comment:
isn't it better to just declare `skinConfig` here and set value in `beforeEach`?

Reply:
or maybe not declare it at all as it is used only in one test

Reply:
\+